### DISCUSSION TITLE
Lyrics

### DISCUSSION THREAD

**Discussion by Dynamite97m** [sayaka-sh/spmp#250] [General]
Make lyrics look like how youtube music does it

**Comment by sayaka-sh** [comment]
I don't use YouTube Music, so could you describe in more detail?

**Comment by Dynamite97m** [comment]
![Screenshot_20240305-113556_YouTube Music](https://github.com/toasterofbread/spmp/assets/162066729/83c54471-d131-413b-83da-2e997cdda46a)
Lyrics like this can you make it

**Comment by Dynamite97m** [comment]
Are you still interested in changing how lyrics look like in the app
